src/init/starship.ps1

```diff
@@ -63,6 +63,13 @@ function global:prompt {
         $process.StandardOutput.ReadToEnd();
     }
 
+    # Invoke precmd, if specified
+    try {
+        if (Get-Command Invoke-Starship-PreCommand) {
```

Comment:
I think this can fail if the user has set up a different default error action. You could use `Test-Path function:Invoke-Starship-PreCommand` like in your previous commit instead.

Reply:
Will give it a go and update my PR.

Reply:
@davidkna should be ready now.